### DISCUSSION TITLE
Batch rework of own errors

### DISCUSSION THREAD

**Discussion by chrysn** [streetcomplete/StreetComplete#4882] [Q&A]
I've noticed that I've subtly erred on several edits I've made to OSM via StreetComplete; those were made across several days and already uploaded.

Short of going full just-write-a-shell-script through the OSM logs, is there any easy way to change all occurrences where I marked a house as "semi-detached" as "house", to undo these changes, or even to show them on my map for revisiting?

**Comment by matkoniecz** [comment]
Overpass Turbo allows you to search for edits where you are last person to edit OSM object and it matches specific tags. If you need more hints/instructions feel free to post here for more help.
